spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,59 @@
 require 'rails_helper'
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it {should validate_presence_of(:title)}
+    it {should validate_length_of(:title).is_at_most(255)}
+    it {should validate_presence_of(:body)}
+    it {should validate_length_of(:body).is_at_most(4000)}
+    context 'for state' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.